experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -388,6 +393,18 @@ cs = CoupledSimulation{FT}(
     diagnostics,
 );
 
+#=
+## Restart component model states if specified
+=#
+restart_dir = nothing
+restart_t = nothing
```

Comment:
Are these temporary assignments / determined through CLI args? `restart_t` should not be valid without a `restart_dir` specified right? In `CheckPointer.jl`, `Y .= Y_new` where Y_new is evaluated at `restart_t`; without an appropriate `restart_dir` this should not modify `Y`.

Reply:
No, but I think adding these as `CLI args` makes sense actually - it will make our life easier when testing on the cluster. :-)